buildstockbatch/postprocessing.py

```diff
@@ -214,8 +216,9 @@ def clean_up_results_df(df, cfg, keep_upgrade_id=False):
     return results_df
 
 
-def get_cols(fs, filename):
-    with fs.open(filename, 'rb') as f:
+def get_cols(fs, path, filename):
+    filepath = Path(path) / filename
+    with fs.open(filepath, 'rb') as f:
```

Comment:
Yeah, this could break things depending on which platform you're using. (I'm thinking Windows with the `\` instead of the `/`.) For compatibility with using with on S3, we need to have all file access be done through the [fsspec](https://filesystem-spec.readthedocs.io/en/latest/index.html) object `fs`. Path joining is a [known deficiency](https://github.com/fsspec/filesystem_spec/issues/747) with that library. It'd probably be best to just do this as a string. 

```suggestion
    filepath = f"{path}/{filename}"
    with fs.open(filepath, 'rb') as f:
```

Reply:
Ha, I thought I was being more careful by using pathlibs path joining than using string "/" join.